src/asmcnc/comms/router_machine.py

```diff
@@ -2067,7 +2075,11 @@ def probe_z_detection_event(self, z_machine_coord_when_probed):
         self.s.write_command('G10 L20 P1 Z' + str(self.z_touch_plate_thickness))
         self.s.write_command('G4 P0.5') 
         Clock.schedule_once(lambda dt: self.strobe_led_playlist("datum_has_been_set"), 0.5)
-        self.zUp()    
+        if self.fast_probing:
+            self.jog_relative('Z', 5, 750)
+            self.fast_probing = False
```

Comment:
This line actually isn't needed because it gets set before the movement starts anyway :)

Reply:
I actually did this on purpose for future proofing in case this variable needs to be accessed from elsewhere. And even if it's never needed, to me it still feels a little weird to have a router machine variable that doesn't accurately represent the current state 😅